bin/localnet/deployer/deployer.go

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright (C) 2017 The "MysteriumNetwork/node" Authors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"github.com/MysteriumNetwork/payments/cli/helpers"
+	"github.com/MysteriumNetwork/payments/mysttoken/generated"
+	generated2 "github.com/MysteriumNetwork/payments/promises/generated"
```

Comment:
Why not `generated69`? Lets add meaningful package name

Reply:
aliased as promises and mysttoken accordingly. This is a good idea to improve payments lib